ocs_ci/ocs/jenkins.py

```diff
@@ -349,6 +352,36 @@ def export_builds_results_to_googlesheet(
              f"test_run_name:{utils.get_testrun_name()}"], 2
         )
 
+    def get_nodes(self, node_type='worker', num_of_nodes=1):
+        """
+        get nodes
+
+        Args:
+            node_type (str): The node type  (e.g. worker, master)
+            num_of_nodes (int): The number of nodes to be returned
+
+        Returns:
+            list: List of compute node names
+        """
+        if node_type == 'master':
+            nodes_drain = [node.name for node in get_typed_nodes(node_type=node_type, num_of_nodes=num_of_nodes)]
+        elif node_type == 'worker':
+            pod_objs = []
+            for project in self.projects:
+                pod_names = get_pod_name_by_pattern(
+                    pattern='jenkins', namespace=project
+                )
+                for pod_name in pod_names:
+                    pod_objs.append(
+                        get_pod_obj(name=pod_name, namespace=project)
+                    )
```

Comment:
I think we can use here list comprehension: `pod_objs = [get_pod_obj(name=pod_name, namespace=project) for pod_name in pod_names]` instead of for loop